authors/apps/articles/models.py

```diff
@@ -129,3 +137,31 @@ class Favorite(models.Model):
         related_name='favorites')
     article_id = models.ForeignKey(
         Article, on_delete=models.CASCADE, related_name='favorites')
+
+
+class Rating(models.Model):
```

Comment:
Please include a string representation of this class, if there is no string in the body, return a string concatenated with the stringified version of this class, for example: "this is rating str(self.id)